ledger/src/snapshot_utils.rs

```diff
@@ -146,51 +240,85 @@ pub fn add_snapshot<P: AsRef<Path>>(snapshot_path: P, bank: &Bank) -> Result<()>
     fs::create_dir_all(slot_snapshot_dir.clone())?;
 
     // the snapshot is stored as snapshot_path/slot/slot
-    let snapshot_file_path = slot_snapshot_dir.join(get_snapshot_file_name(slot));
+    let snapshot_bank_file_path = slot_snapshot_dir.join(get_snapshot_file_name(slot));
     info!(
         "creating snapshot {}, path: {:?}",
         bank.slot(),
-        snapshot_file_path,
+        snapshot_bank_file_path,
     );
 
-    let snapshot_file = File::create(&snapshot_file_path)?;
-    // snapshot writer
-    let mut snapshot_stream = BufWriter::new(snapshot_file);
-    // Create the snapshot
-    serialize_into(&mut snapshot_stream, &*bank)?;
-    let mut bank_rc_serialize = Measure::start("create snapshot");
```

Comment:
(This is a re-post of  [this outdated comment](https://github.com/solana-labs/solana/pull/7182#discussion_r358152479))

This `Measure` will start to include the processing time of [serializing a Bank](https://github.com/solana-labs/solana/pull/7182/files#diff-0f0af8fcfa1444e1b07d671c5bfa4a69R255), in addition to original `BankRc`. I think including it isn't a problem because those serializations (`Bank` and `BankRc`) are tightly coupled. And, this is referenced by our Grafana's _Time spent in squashing (All)_ graph, so I think it's OK to do so, in that standpoint too, because the time of serializing `Bank` wasn't included, which should be in the graph.

Accordingly, I've renamed to reflect the definition change. 

@carllin  Is there any concern for this measurement definition change? (I'm asking based on [the original author of this `Measure`](https://github.com/solana-labs/solana/pull/6047/files#diff-ff88facee39736079f15b8937943a82eR144)) :)

Reply:
@ryoqun yup that should be no problem, serializing the other parts of the bank should be negligible anyways 😃 

Looks like you updated the testnet-monitor.json as well, nice!